src/library/basetracktablemodel.cpp

```diff
@@ -463,6 +467,40 @@ bool BaseTrackTableModel::setData(
     return setTrackValueForColumn(pTrack, column, value, role);
 }
 
+QVariant BaseTrackTableModel::composeCoverArtToolTipHtml(
+        const QModelIndex& index) const {
+    // Determine height of the cover art image depending on the screen size
+    unsigned int absHeightOfCoverartTooltip;
+    const QScreen* primaryScreen = getPrimaryScreen();
+    if (primaryScreen) {
+        absHeightOfCoverartTooltip = static_cast<int>(
+                primaryScreen->availableGeometry().height() *
+                kRelHeightOfCoverartTooltip);
+    } else {
+        qWarning() << "Assuming screen height of 800px.";
```

Comment:
If the screen is not accessible we should not display a tool tip. This should never happen.

```
VERIFY_OR_DEBUG_ASSERT(pScreen) {
    return QVariant();
}
```

Reply:
Done!